schemas/rum/error-schema.json

```diff
@@ -177,6 +177,126 @@
                 }
               },
               "readOnly": true
+            },
+            "threads": {
+              "type": "array",
+              "description": "Description of the each thread in the process when error happened.",
```

Comment:
Should we remove `each` here to make it consistent with the description in `error-schema.json` ?